todo-app/src/todoBox.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+class todoBox extends React.Component {
+    constructor(props) {
+    super(props);
+    this.state = {
+        clicked: this.props.done
+    }
+    this.handleChange = this.handleChange.bind(this);
+}
+handleChange(event) {
+    let clicked = !this.state.clicked
+    this.setState({
+        clicked: clicked
+    })
+}
+changeStyle() {
+    let style = {color: 'red'}
+    if (this.state.clicked) {
+        style = {color: 'greenyellow'}
+    }
+    return style
+}
+getTodoStatus () {
+    let doneOrNot = 'Not done'
+    if (this.state.clicked) {doneOrNot = "Done"}
+    return doneOrNot
+}
+
+render() {
+    return (
+        <div>
+        <input type = "checkbox"
+        clicked = {this.state.clicked}
```

Comment:
There is no `clicked` attribute for [`input checkbox`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox) element. Maybe you wanted to use something else?

Reply:
I've checked my code 2 3 times but I think I did mistake I did put some code while we were using last Sunday, reading all code is hard